public/resources/img/icon/touch-emulator-on.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```

Comment:
Please run this SVG through [svgo](https://www.npmjs.com/package/svgo) so that all the "bits that do nothing", comments, nonstandard attributes, etc. get removed